mappings/net/minecraft/server/network/ServerPlayerEntity.mapping

```diff
@@ -1,12 +1,21 @@
 CLASS net/minecraft/class_3222 net/minecraft/server/network/ServerPlayerEntity
+	FIELD field_13963 healthScore F
+	FIELD field_13965 levelScore I
```

Comment:
Maybe `last<X>Score` ? Considering it is used specifically to check whether the scoreboard needs updating.

Reply:
Sure, that'll work.